samples/ImplicitFlow/AuthorizationServer/Startup.cs

```diff
@@ -0,0 +1,156 @@
+                .AddEntityFrameworkStores<ApplicationDbContext, Guid>()
+                .AddDefaultTokenProviders();
+
+            // Register the OpenIddict services, including the default Entity Framework stores.
+            services.AddOpenIddict<ApplicationUser, IdentityRole<Guid>, ApplicationDbContext, Guid>()
+
+                // Enable the authorization, logout, userinfo, and introspection endpoints.
+                .EnableAuthorizationEndpoint("/connect/authorize")
+                .EnableLogoutEndpoint("/connect/logout")
+                .EnableUserinfoEndpoint("/connect/userinfo")
+                .EnableIntrospectionEndpoint("/connect/introspect")
+
+                // Note: the Mvc.Client sample only uses the authorization code flow but you can enable
+                // the other flows if you need to support implicit, password or client credentials.
+                .AllowImplicitFlow()
+
+                // During development, you can disable the HTTPS requirement.
+                .DisableHttpsRequirement()
+
+                // Register a new ephemeral key, that is discarded when the application
+                // shuts down. Tokens signed using this key are automatically invalidated.
+                // This method should only be used during development.
+                .AddEphemeralSigningKey();
+
+            services.AddCors();
+            services.AddMvc();
+
+            services.AddTransient<IEmailSender, AuthMessageSender>();
+            services.AddTransient<ISmsSender, AuthMessageSender>();
+        }
+
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILogger<Startup> logger)
+        {
+            // Set the Environment e.g. via PowerShell> $env:ASPNETCORE_ENVIRONMENT="Development"
+            logger.LogInformation(env.EnvironmentName);
+
+            app.UseIdentity();
+            app.UseOAuthValidation();
+
+            app.UseCors(builder => builder
+                .AllowAnyHeader()
+                .AllowAnyOrigin()
+                .AllowAnyMethod());
+
+            app.UseOpenIddict();
+            app.UseMvcWithDefaultRoute();
+
+            // In a production app, seed this in a setup tool.
+            SeedDatabase(app, env);
+        }
+
+        private void SeedDatabase(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            var options = app
+                .ApplicationServices
+                .GetRequiredService<DbContextOptions<ApplicationDbContext>>();
+
+            using (var context = new ApplicationDbContext(options))
+            {
+                // drop and recreate
+                context.Database.EnsureDeleted();
+                context.Database.EnsureCreated();
+
+                if (!context.Applications.Any())
+                {
+                    var aureliaOidcHost = env.IsDevelopment()
+                            ? "http://localhost:9000"
+                            : "https://zamboni-app.azurewebsites.net";
```

Comment:
This address is specific to your deployment, please remove this variable and use `http://localhost:9000`.